src/avif.c

```diff
@@ -57,9 +57,11 @@ void avifGetPixelFormatInfo(avifPixelFormat format, avifPixelFormatInfo * info)
             break;
 
         case AVIF_PIXEL_FORMAT_YUV400:
+            info->monochrome = AVIF_TRUE;
+            // 4:0:0 is stored as monochrome and the nonexistent chroma is considered as subsampled in
```

Comment:
Nit: "stored as monochrome" sounds a little strange. I suggest changing "4:0:0 is stored as monochrome" to "4:0:0 is monochrome", or perhaps just delete it?